cmd/rancher-desktop-guestagent/main.go

```diff
@@ -210,6 +216,12 @@ func main() {
 	}
 
 	if err := group.Wait(); err != nil {
+		if *enableDocker && !*enablePrivilegedService {
+			if err := iptables.LoopbackRules(iptables.DeleteFromChain); err != nil {
```

Comment:
Same here. Actually, this should be a `defer` next to the previous block, because…

Reply:
Yes, I had this as a defer statement initially but Linter was screaming at me. I guess I will put it back.